example/aks/main.tf

```diff
@@ -0,0 +1,114 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.44.0"
+    }
+  }
+  required_version = ">=0.14.8"
+}
+
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_subscription" "current" {
+}
+
+resource "random_string" "random" {
+  length  = 12
+  upper   = false
+  number  = false
+  special = false
+}
+
+module "subscription" {
+  source = "github.com/Azure-Terraform/terraform-azurerm-subscription-data.git?ref=v1.0.0"
+  subscription_id = data.azurerm_subscription.current.subscription_id
+}
+
+module "naming" {
+  source = "github.com/Azure-Terraform/example-naming-template.git?ref=v1.0.0"
+}
+
+module "metadata" {
+  source = "github.com/Azure-Terraform/terraform-azurerm-metadata.git?ref=v1.1.0"
+
+  naming_rules = module.naming.yaml
+
+  market              = "us"
+  project             = "https://github.com/Azure-Terraform/terraform-azurerm-virtual-network/tree/master/example/bastion"
+  location            = "eastus2"
+  environment         = "sandbox"
+  product_name        = random_string.random.result
+  business_unit       = "infra"
+  product_group       = "contoso"
+  subscription_id     = module.subscription.output.subscription_id
+  subscription_type   = "dev"
+  resource_group_type = "app"
+}
+
+module "resource_group" {
+  source = "github.com/Azure-Terraform/terraform-azurerm-resource-group.git?ref=v1.0.0"
```

Comment:
Suggest updating to current version of resource-group module, I think we're at 2.0.0.